packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -269,6 +275,23 @@ export async function pushCommand(
 	}
 }
 
+export async function checkImagePlatform(
```

Comment:
Since it's only used in this file (for now) let's keep it private.

```suggestion
async function checkImagePlatform(
```